docs/API/NuGet-Protocols.md

```diff
@@ -13,77 +13,89 @@ ms.assetid: ba1d9742-9f1c-42ff-8c30-8e953e23c501
 
-This protocol specifies usage of verify-scope keys to interact with services other than nuget.org, to validate a package against a nuget.org account. 
+This protocol specifies usage of verify-scope keys to interact with services other than nuget.org, to validate a
+package against a nuget.org account. 
 
-Validation ensures that the user-created API keys are used only with nuget.org, and that other verification or validation from a third-party service is handled through a one-time use verify-scope keys. These verify-scope keys can be used to validate that the package belongs to a particular user (account) on nuget.org.
+Validation ensures that the user-created API keys are used only with nuget.org, and that other verification or
+validation from a third-party service is handled through a one-time use verify-scope keys. These verify-scope keys can
+be used to validate that the package belongs to a particular user (account) on nuget.org.
 
 ### Client requirement
 
-If a client interacts with other services and needs to validate whether a package belongs to a particular user (account), it should use this protocol and use the verify scope keys and not the API keys from nuget.org.
-
 Clients are required to pass the following header when they make API calls to **push** packages to nuget.org:
 
 ```
 X-NuGet-Protocol-Version: 4.1.0
 ```
 
-### Protocol usage details
+Note that the pre-existing `X-NuGet-Client-Version` header has the same purpose but is now deprecated and should no
+longer be used.
+
+The **push** protocol itself is described in the documentation for the
+[`PackagePublish` resource](v3/package-publish-resource.md).
+
+If a client interacts with externals services and needs to validate whether a package belongs to a particular user
+(account), it should use the following protocol and use the verify-scope keys and not the API keys from nuget.org.
+
+### API to request a verify-scope key
 
-#### API to request a verify scope key
 This API is used to get a verify-scope key for a nuget.org author to validate a package owned by him/her.
 
 ```
-POST api/v2/package/create-verification-key/{id}/{version}
+POST api/v2/package/create-verification-key/{ID}/{VERSION}
 ```
 
-Request parameters:
+#### Request parameters
 
-| Name | In | Type | Notes|
-|:------------- |:-------------|:----|:-----|
-| id    | URL | string | Required: the package identidier for which theverify scope key is requested. |
-| version | URL | string | Optional: package version|
-| X-NuGet-ApiKey | Header | string | Required: for example, `X-NuGet-ApiKey={USER_API_KEY}` |
+Name           | In     | Type   | Notes
+-------------- | ------ | ------ | ----- 
+ID             | URL    | string | Required: the package identidier for which theverify scope key is requested
+VERSION        | URL    | string | Optional: package version
+X-NuGet-ApiKey | Header | string | Required: for example, `X-NuGet-ApiKey: {USER_API_KEY}`
 
-Response:
+#### Response
```

Comment:
Can an example response be added?

Reply:
The response body is empty. Added status codes.